### PR TITLE
libgetdata: optional python and php bindings

### PR DESCRIPTION
**Pull request by arahlin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Options to build python2/3 and php bindings

### CODE REVIEW

File: Formula/libgetdata.rb [hunk 1]
```diff
@@ -25,21 +28,38 @@ class Libgetdata < Formula
   depends_on "gcc" => :optional
   depends_on "libzzip" => :optional
   depends_on "perl" => :optional
+  depends_on "php" => :optional
+  depends_on "python" => :optional
+  depends_on "python@2" => :optional
   depends_on "xz" => :optional
 
+  if build.with?("python") && build.width?("python@2")
```
Comment (kabel) [inline Formula/libgetdata.rb]: Typo. `build.width?`

Comment (arahlin) [reply Formula/libgetdata.rb]: whoops!

**Comment by fxcoudert** [comment]
Analytics show that this formula is little used, and existing options are not actually used. We should rather remove the existing options that add new ones.

```
install events in the last 30 days for libgetdata
================================================================================
1 | libgetdata                                                    | 11 |  78.57%
2 | libgetdata --with-python                                      |  3 |  21.43%
================================================================================
Total                                                             | 14 |    100%
================================================================================
```

**Comment by arahlin** [comment]
To me this formula is useless without the bindings.  Perhaps we can let the configure script figure out which bindings to build based on what it finds installed?  I would leave the perl and python options in place, since they are specific to the way Homebrew packages these libraries.

**Pull request closed without merging**